data/remnant/remnant missions.txt

```diff
@@ -2208,3 +2208,655 @@ mission "Remnant: Face to Maw 2B"
+			`	This is one of the aspects that set prefects apart: They are Remnant who have demonstrated excellence in their field, sound judgment across a variety of fields, and an affinity for balancing a multitude of demands on Remnant resources. Humility is also a requirement, as they must always recognize that while their particular combination of skills may be unique, there is always someone else who is better than them at any one thing. As such, Prefects are the adjudicators in determining the priority of our various tasks, and the leaders who keep the Remnant unified.`
+			choice
+				`	"Thanks for the explanation!"`
+					goto acceptance
+				`	"I thought prefects were specialists?"`
+			`	"Well, they are and are not. Most prefects are assigned particular tasks that they are expected to devote themselves to. Often, this task relates to their area of expertise, but it is not always the case. Generally speaking, a prefect is considered to be the ultimate authority on tasks that they have been assigned to, but they are expected to take into account the information and suggestions from other prefects and especially the specialists in the field or fields relevant to the task.`
+			`	"Given your own skills, I suspect that the prefects you are most likely to interact with would be Prefect Chilia who has the task of co-ordinating the military defense of the Remnant; and Prefect Taely who has the task of integrating our latest innovations into usable ships."`
+			choice
+				`	"Thanks! That was more detail than I was expecting."`
+				`	"Okay, that satisfied my curiosity for now."`
+			label acceptance
+			`	"Excellent. You deal with the Korath, and I will file a report about this threat. Please inform me immediately once you have dealt with the threat." He pauses, then continues, "Oh, and remember to record the battle. It may be useful to help understand what they are doing there."`
+				accept
+	npc kill
+		government "Korath"
+		system "Nenia"
+		personality forbearing timid uninterested surveillance staying
+		ship "Korath Raider" "Korath Raider (Ember)"
+			cargo "void sprite" 5 20
+		dialog "You have destroyed the Korath Vessel. You can now return to <planet> to collect your payment."
+	on complete
+		event "korath hunting"
+		conversation
+			`When you return to Aventine, you find Plume waiting for you. He eagerly reviews the recording of the combat. "Thank you for eliminating this hunter..." he is interrupted by a flashing light on his commlink. He briefly reads something, then looks back up to you. "Apparently something about the Korath hunting the void sprites has concerned the defence prefect. Chilia has requested that you meet with him on Caelian immediately."`
```

Comment:
```suggestion
			`When you return to Aventine, you find Plume waiting for you. He eagerly reviews the recording of the combat. "Thank you for eliminating this hunter-" he is interrupted by a flashing light on his commlink. He briefly reads something then looks back up to you. "Apparently something about the Korath hunting the void sprites has concerned the defense prefect. Chilia has requested that you meet with him on Caelian immediately."`
```
I like a dash instead of an ellipsis to signify interruption, but it's personal preference.